package.json

```diff
@@ -24,6 +24,7 @@
   },
   "homepage": "https://github.com/bindoctor/BinDr#readme",
   "dependencies": {
+    "dotenv": "^8.0.0",
```

Comment:
What is the dotenv used for?

Reply:
dotenv is used for managing environment variables. We were adding to the project so that we can store api keys in a .env file that is not committed to our repository.

Reply:
Cool I remember using the dotenv in ruby I think and now it came back that it was with the .env file! great cheers!